tox.ini

```diff
@@ -1,13 +1,11 @@
 [tox]
-envlist = pep8,pylint,{pypy3,py36,py37,py38,py39,py310}-pymongo-pyexecjs,py310,py310-{pymongo,pyexecjs},py39-{pymongo3,pymongo361}-pyexecjs,py310-pymongo-pyexecjs-mongodb4
+envlist = pep8,pylint,{pypy3,py38,py39,py310}-pymongo-pyexecjs,py310,py310-{pymongo,pyexecjs},py39-{pymongo3,pymongo361}-pyexecjs,py310-pymongo-pyexecjs-mongodb4
 
 [testenv]
-passenv = CI TRAVIS TRAVIS_* TEST_MONGO_HOST MONGODB
+passenv = CI, TRAVIS, TRAVIS_*, TEST_MONGO_HOST, MONGODB
 setenv =
   EXECJS_RUNTIME = Node
 basepython =
-  py36: python3.6
-  py37: python3.7
   py38: python3.8
   py39: python3.9
   py310: python3.10
```

Comment:
Please add python 3.11 and python 3.12 (or remove them from the other places).

Reply:
The point is to eventually replace `tox` with `hatch`, but I can certainly add the additional Python versions for now